src/slider/styles/slider.css

```diff
@@ -0,0 +1,193 @@
+@import '../../common/styles/variables.css';
+
+.root {
+	display: block;
+	font: var(--base-font);
+	margin-bottom: var(--line-spacing);
+}
+
+.inputWrapper {
+  position: relative;
+}
+
+.track {
+	background-color: var(--dojo-light-gray);
+	border: 1px solid var(--input-border-color);
+	border-radius: var(--rounded-corners);
+	height: 0.75em;
+	position: relative;
+	width: 100%;
+}
+
+.fill {
+	background-color: var(--input-highlight-color);
+	border-radius: var(--rounded-corners);
+	bottom: 0;
+	content: "";
+	left: 0;
+	position: absolute;
+	top: 0;
+}
+
+.thumb {
+	background-color: var(--input-highlight-color);
+	border-radius: 50%;
+	box-shadow: inset 0 0 3px 0 var(--dojo-shade-gray);
```

Comment:
I considered it, but we already have two box shadows in variables already :P. I can't use either of those b/c the thumb is too small for them. I think it might actually be cleaner overall to keep this one specific to Slider, at least until we get official styles for all this.